### ISSUE TITLE
My Best Times on Challenges do not Save After Closing the Game

### ISSUE THREAD

**Issue by Cameron322** [treeform/istrolid-bugs#24]
Whenever I complete a challenge my best time will show up in the bottom of the square that was completed. But whenever I close the game out or press Quit Game and then re-open the game all of my best times will be gone (even though pressing Quit Game says that it saves all the data). I've tried everything and can't seem to figure out why this is. Please help. Also is anyone else experiencing this problem?

**Comment by bokken12** [comment]
It's not just you the saving of challenges has never been quite right. It's a bug that generally doesn't annoy too many people so has been able to stay there for a while but should be fixed.

**Comment by Cameron322** [comment]
Yeah, I've heard that most people don't do the challenges so this would not affect them but I do think it would be fun to be able to test different strategies and fleets on the different challenges. I really hope that they will finally fix this issue because without being able to see your best time it becomes less useful and less fun.

**Comment by treeform** [comment]
We should redo how the challenges work.
